components/ws-manager/pkg/manager/metrics.go

```diff
@@ -386,25 +365,3 @@ func newSubscriberQueueLevelVec(m *Manager) *timeoutSettingsVec {
 		desc:    desc,
```

Comment:
Thank you for uncovering this pre-existing bug :)
The `subscriber_queue_level` metric should use `subscriberQueueLevelVec` instead of `timeoutSettingsVec`. That also explains the bogus values we've read from this metric 🤦

Reply:
fixed